.credo.exs

```diff
@@ -0,0 +1,18 @@
+%{
+  configs: [
+    %{
+      name: "default",
+      files: %{
+        included: ["lib/", "test/"]
+      },
+      strict: true,
+      color: true,
+      checks: %{
+        disabled: [
+          # this means that `TagTODO` will not run
+          {Credo.Check.Design.TagTODO, []}
```

Comment:
While this is perfectly fine, you'd want to consider if you might prefer having warnings about TODOs but just prevent them from failing Credo checks:

```elixir
enabled: [
  {Credo.Check.Design.TagTODO, [exit_status: 0]}
]
```